components/user-list/UserList.js

```diff
@@ -18,51 +19,44 @@ export default class UserList extends Component {
     this.renderItem = this.renderItem.bind(this)
   }
 
-  keyExtractor(_, index) {
-    return index.toString()
-  }
-
-  sectionData() {
-    const { users } = this.state
-
-    const sections = createSectionData(users)
-
-    return sections.filter((section) => section.data.length !== 0)
-  }
-
   renderItem({ item }) {
     const { navigation } = this.props
     const { products } = this.state
 
     return (
-      <ListItem
-        button
-        title={item.name}
-        leftAvatar={{ source: { uri: item.avatar } }}
+      <TouchableOpacity
+        style={styles.item}
         onPress={() => {
           navigation.navigate('ProductList', {
             user: item,
             products,
           })
         }}
-      />
+      >
+        <View style={styles.avatarContainer}>
+          <Avatar
+            rounded
+            size={120}
```

Comment:
I don't think we should use hard-coded size values. We could replace by  `large`  to keep compatibility with smaller devices.
```suggestion
            size="large"
```

Reply:
That makes sense.